README.md

```diff
@@ -15,15 +15,41 @@ You need to install Rust and some dependencies.
 You can check os dependent installation [here](https://tauri.studio/v1/guides/getting-started/prerequisites/).
 
 ### Setup the project
-Clone the project, go into it and install the Javascript dependencies.  
-```git clone git@github.com:Brawdunoir/dionysos-client.git```  
-```cd dionysos-client```  
-```pnpm i```
+Clone the project
+```
+git clone git@github.com:Brawdunoir/dionysos-client.git
+```
+CD Into the project directory
```

Comment:
While you removed the capitalize letter on pnpm, which I can get as it's an acronym, you capitalized the whole command and the word after and I don't think it looks consistent

Reply:
Just remove all uppercase letters, I think it's fine